docs/DEVELOPING.md

```diff
@@ -343,6 +343,139 @@ $ make ftest NAME=azure_blob_storage
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Oracle Database details](#gather-oracle-database-details)
+2. [Configure Oracle Database connector](#configure-oracle-database-connector)
+
+#### Gather Oracle Database details
+
+Collect the information that is required to connect to your Oracle Database:
+
+- The server host address where the Oracle Database is hosted.
+- The port where the Oracle Database is hosted.
+- The username the connector will use to log in to the Oracle Database.
+- The password the connector will use to log in to the Oracle Database.
+- The database name where the connector will query data.
+
+#### Configure Oracle Database connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `host`
+
+The server host address where the Oracle Database is hosted. Default value is `127.0.0.1`. Examples:
+
+  - `192.158.1.38`
+  - `demo.instance.demo-region.demo.service.com`
+
+##### `port`
+
+The port where the Oracle Database is hosted. Default value is `9090`. Note: For a secured connection, a user needs to configure a SSL port.
+
+  - `5432`
+  - `9090`
+
+##### `username`
+
+The username of the account for the Oracle Database. Default value is `admin`.
+
+##### `password`
+
+The password of the account to be used for the Oracle Database. Default value is `Password_123`.
+
+##### `database`
+
+Name of the Oracle database. Default value is `xe`. Examples:
+
+  - `employee_database`
+  - `customer_database`
+
+##### `tables`
+
+Comma-separated list of tables. The Oracle Database connector will fetch data from all tables present in the configured database, if the value is `*`. Default value is `*`. Examples:
+
+  - `table_1, table_2`
+  - `*`
+
+##### `fetch_size`
+
+Number of rows to fetch on each request to Oracle Database. Default value is `50`.
+
+##### `retry_count`
+
+The number of retry attempts after failed request to Oracle Database. Default value is `3`.
+
+##### `oracle_protocol`
+
+The protocol which the connector uses to establish a connection. Default value is `TCP`. Note: For a secured connection, a user needs to use `TCPS`.
+
+##### `oracle_home`
+
+Path of the Oracle home directory to run connector with thick mode for secured connection. Note: In case of unsecured connection, keep `oracle_home` field empty.
+
+##### `wallet_configuration_path`
+
+Path of the oracle configuration files. Only applicable when configuration files are not at the default location. Note: If Wallet configuration files are located at default location, keep `wallet_configuration_path` field empty.
```

Comment:
By "oracle configuration files", are we referring to standard Oracle database configuration files? Or are we referring to Oracle Wallet (e.g. https://docs.oracle.com/middleware/1213/wls/JDBCA/oraclewallet.htm) which is a way to secure credentials in Oracle across domains?

Reply:
In the subsequent section, we talk about setting a secured connection with Oracle with instructions to store the wallet at `$ORACLE_HOME/ssl_wallet`.  Does it mean that when we say "Only applicable when configuration files are not at the default location", the default location is `$ORACLE_HOME/ssl_wallet`? If so, it can be helpful to include that in the description of the setting.

Reply:
By wallet_configuration_path it contains files like `sqlnet.ora` and `tnsadmin.ora`  which is by default available at `$ORACLE_HOME/network/admin`. Added description for default path.